worker/tests/unit/productionDeployJob.unit.test.js

```diff
@@ -15,23 +16,29 @@ const testPayloadWithRepo = {
 describe('Test Class', () => {
   // Dont actually reset the directory and dont care about the logging
   beforeAll(() => {
+    try {
     workerUtils.resetDirectory = jest.fn().mockResolvedValue();
     workerUtils.logInMongo = jest.fn().mockResolvedValue();
-    const githubJob = new GitHubJob(testPayloadWithRepo);
-
-    githubJob.buildRepo = jest.fn().mockReturnValueOnce({
+    this.githubJob = new GitHubJob(testPayloadWithRepo);
+    this.githubJob.buildRepo = jest.fn().mockReturnValueOnce({
       status: 'success',
       stdout: null,
       stderr: null,
     });
+    this.logger = new Logger(this.githubJob)
+    this.logger.filterOutputForUserLogs = jest.fn().mockReturnValueOnce({
+      status: 'success',
+      stdout: null,
+      stderr: null,
+    });
+    this.logger.save = jest.fn().mockReturnValueOnce({});
     jest.useFakeTimers();
+  } catch(ex) {
+    console.log(ex)
```

Comment:
is `ex` an error? If so, let's rename it to er` or `err`

Reply:
Also, should we not be throwing the error? Or is there reason not to here?

Reply:
No i should remove it, it was a debug residue